app/views/users/show.html.erb

```diff
@@ -1 +1,15 @@
-<h1>welcome to the user page</h1>
+<div class="user-page-container">
+  <%= render "shared/user_details", user: @user %>
```

Comment:
- [ ] @Ibtisam34 kindly change the rendered  link to match with the shared file naming, also in your design kindly ensure the posts section matches the width matches with the other page section 
![image](https://user-images.githubusercontent.com/61408860/225054920-eb98b5d1-4944-4873-b1e3-9ff4f86cb0da.png)